hw/ip_templates/flash_ctrl/dv/env/flash_ctrl_env.core

```diff
@@ -15,6 +15,8 @@ filesets:
       - lowrisc:dv:flash_phy_prim_agent
       - lowrisc:ip_interfaces:flash_ctrl_pkg
       - lowrisc:constants:top_pkg
+      # TODO: prim_pkg_legacy is deprecated
+      - lowrisc:prim:prim_pkg_legacy
```

Comment:
Maybe good to put this in a separate commit, just to separate it from the prepend changes.